### PR TITLE
#13480 :bug: solved the flip bug in screen share video

### PR DESCRIPTION
**Pull request by fatemehmarzoughi**
There was an issue with the screen share in which after changing the virtual background the screen share video tag had a scale with a -1 value, I edited this and added the `!isScreenShare()` condition before it. Now the issue doesn't exists anymore :)

### CODE REVIEW

File: modules/UI/videolayout/VideoContainer.js [hunk 1]
```diff
@@ -497,7 +497,7 @@ export class VideoContainer extends LargeContainer {
 
             const flipX = stream.isLocal() && this.localFlipX && !this.isScreenSharing();
 
-            this.video.style.transform = flipX ? 'scaleX(-1)' : 'none';
+            this.video.style.transform = flipX && !this.isScreenSharing() ? 'scaleX(-1)' : 'none';
```
Comment (damencho) [inline modules/UI/videolayout/VideoContainer.js:500]: Hum, but `!this.isScreenSharing()` is taken into account two lines above? I don't see the change with this PR?

**Comment by fatemehmarzoughi** [comment]
@damencho Sorry for the previous mistake, I just updated the pull request
Thanks in advanced

File: modules/UI/videolayout/VideoContainer.js (hunk 1 above)
Comment (fatemehmarzoughi) [reply modules/UI/videolayout/VideoContainer.js]: @damencho Sorry for the previous mistake, I just updated the pull request
Thanks in advanced

**Comment by saghul** [comment]
The change LGTM now.

**Review by saghul — APPROVED** [review]

**Comment by damencho** [comment]
Jenkins test this please.

**Pull request merged**